src/main/scala/io/qbeast/internal/commands/ConvertToQbeastCommand.scala

```diff
@@ -75,14 +76,20 @@ case class ConvertToQbeastCommand(
       fileFormat match {
         // Convert parquet to delta
```

Comment:
Change description to: Convert parquet to $DEFAULT_TABLE_FORMAT...

Reply:
fixed